packages/pg-v5/test/unit/commands/backups/restore.unit.test.js

```diff
@@ -13,7 +13,7 @@ let restoringText = () => {
   return process.stderr.isTTY ? 'Restoring... pending\nRestoring... done\n' : 'Restoring... done\n'
 }
 
-const shouldRestore = function (cmdRun) {
+const shouldRestore = function () {
```

Comment:
This file was changed to allow migration runs. It also makes the code less strange.